src/bindings/python/src/pyopenvino/core/remote_context.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include <openvino/core/any.hpp>
+#include <openvino/core/type/element_type.hpp>
+#include <openvino/runtime/intel_gpu/properties.hpp>
+#include <openvino/runtime/intel_gpu/remote_properties.hpp>
+#include <openvino/runtime/remote_context.hpp>
+#include <openvino/runtime/tensor.hpp>
+
+#include "openvino/core/except.hpp"
+#include "pyopenvino/core/remote_tensor.hpp"
+
+#ifdef PY_ENABLE_GPU
+#ifndef _WIN32
+#    include <va/va.h>
```

Comment:
we can use:
https://github.com/openvinotoolkit/openvino/blob/851bdb404c65396f0efb0d5e590ee2f279df4db5/src/plugins/intel_gpu/include/va/va.h#L5-L7
otherwise, who installs `va/va.h` ? Build machine can miss this package

Reply:
Use of raw `void *` instead.